config.yml

```diff
@@ -1,6 +1,13 @@
 version: 1
 build:
   main: cmd/poktrolld
+  ldflags:
+    # The dao reward address SHOULD match that of the "pnf" below (i.e. `make poktrolld_addr ACC_NAME=pnf`).
+    # This is the address that will receive the dao/foundation rewards during claim settlement (global mint TLM).
+    # DEV_NOTE: This is a temporary solution which allows us to configure the dao reward address between networks
+    # (e.g. localnet, testnet, etc.)
+    # TODO_TECHDEBT: Make this a tokenomics module parameter.
```

Comment:
This is pretty important and I'd rather do it sooner rather than later.

Would prefer that we add a PR (before this) and avoid this TECHDEBT altogether. Wouldn't that be easier/faster/cleaner?

Reply:
@bryanchriswhite 

1. How long would you say it takes you to add a new param right now?
2. Do you think there's a way (a need?) to streamline/automate [1] in some way?

[1] https://dev.poktroll.com/develop/developer_guide/adding_params?_highlight=para

Reply:
As discussed offline:

> How long would you say it takes you to add a new param right now?

I think I can bang out several in a day - the docs have gotten pretty solid after dog-fooding them multiple times. It seems like we have 9 that we want to add at the moment.

> Do you think there's a way (a need?) to streamline/automate [1] in some way?

1. I think it's definitely possible.
2. I don't see a trivial way to do it. It seems to me that it requires a similar level of complexity as that of the params doc generator or the protobuf option detector scripts (i.e. AST walking + templating), which will take longer to implement and debug than just doing it manually, in my estimation. I think an ideal automated solution would extend ignite somehow, rather than writing a standalone script, in order to improve ecosystem alignment.

Reply:
This param is added in #922.